packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -367,6 +367,7 @@ async function startCompass(opts: StartCompassOptions = {}): Promise<Compass> {
     waitforInterval: 100, // default is 500ms
   };
 
+  process.env.COMPASS_CLUSTERED_COLLECTIONS = 'true';
```

Comment:
Always enabling the feature flag in e2e tests so we can test this. Wondering if we shouldn't make that policy, actually.

Reply:
I'm thinking it would be better to not have feature flags enabled in end to end tests. That way we know what we're releasing is what we've tested. I wouldn't be opposed to running tests again with flags enabled if we want to double up some tests, but I think we run a risk by enabling feature flags in end to end tests.

Reply:
I added these questions to the Technical Alignment Sync Notes doc for future discussion.